packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -40,13 +41,13 @@ export const SimpleDocument: FC<Props> = ({
   let html,
     passage: QueryResultPassage | null = null;
   if (document) {
-    //Json object usually don't have enough info to allow us to determine which field to display
-    //Unless there is passage pointing to a specific field.
-    const isJsonType = get(document, 'extracted_metadata.file_type') === 'json';
-    if (isJsonType && (!highlight || !isPassage(highlight))) {
+    // JSON files will default to displaying the specified body field, `text` field, or passage highlighting field,
+    // Otherwise an error is shown
+    const isJsonType = getDocumentType(document) === 'json';
```

Comment:
Shall we store these strings ( 'json', 'csv ' etc) somewhere so that it can be used across these files? (here and in DocumentPreview.tsx)